### PR TITLE
CDAP-15821 Whole-file-ingest plugin validation - package rename

### PR DESCRIPTION
**Pull request by mlozbin-cybervisiontech**
JIRA: https://issues.cask.co/browse/CDAP-15821
WIKI: https://wiki.cask.co/display/CE/Guidelines+on+Plugin+Validation

In scope of this PR:
- migrated to CDAP core 6.1.0- SNAPSHOT
- renamed packages to io.cdap

### CODE REVIEW

**Comment by mlozbin-cybervisiontech** [comment]
@albertshau could you, please, take a look.

File: pom.xml [hunk 1]
```diff
@@ -159,7 +165,7 @@
             <Embed-Transitive>true</Embed-Transitive>
             <Embed-Directory>lib</Embed-Directory>
             <!--Only @Plugin classes in the export packages will be included as plugin-->
-            <_exportcontents>co.cask.hydrator.*</_exportcontents>
+            <_exportcontents>io.cdap.plugin.file.ingest.*,io.cdap.plugin.common.*,com.google.common.base.*</_exportcontents>
```
Comment (albertshau) [inline pom.xml]: why were io.cdap.plugin.common and com.google.common.base added? Was it failing without those?

Comment (mlozbin-cybervisiontech) [reply pom.xml]: I've added this because of warnings from maven-bundle-plugin. It makes sense because we use those classes and dependencies are in 'compile' scope.
So I have several ways to avoid it:
1. change scope to 'provided' for hydrator-common and guava. In this case I'm not sure both are provided by CDAP.
2. Ignore warnings from maven plugin.

Maybe you have some advice?

Comment (albertshau) [reply pom.xml]: Exports should only be done for packages that contain the actual plugin classes (this is how CDAP discovers the plugins), and for anything required by Hadoop in order to instantiate the InputFormat/OutputFormat. Anything here will be shared across all plugins used in the same pipeline. Multiple plugins exporting the same packages will cause conflicts. This is especially bad for common classes like com.google.common.base.

Basically, you should never add anything here unless you are getting an error because it is not there. Please remove these unless you are seeing errors running a pipeline.

**Review by albertshau — APPROVED** [review]
